doc/scripts/pullsource.sh

```diff
@@ -2,7 +2,8 @@
 # Copyright (C) 2019 Intel Corporation.
 # SPDX-License-Identifier: BSD-3-Clause
 
-q="--quiet"
+#q="--quiet"
+q=""
```

Comment:
I'm assuming you changed and kept this modification on purpose?

Reply:
yes.  This script can take a l-o-n-g time on WSL, so it's best it not run quietely.